projects/cashmere-examples/src/lib/modal-overview/modal-overview-example-modal.component.html

```diff
@@ -1,6 +1,12 @@
 <hc-modal>
     <hc-modal-header>Modal Header Title</hc-modal-header>
-    <hc-modal-body>Data: {{ activeModal.data }}</hc-modal-body>
+    <hc-modal-body tight="true">
```

Comment:
`<hc-modal-body tight="true">` will work because `"true" == true`, but technically this should probably be `<hc-modal-body [tight]="true">`

Reply:
Ah that's a really interesting point.  In most cases I try to add the `parseBooleanAttribute` utility function to params that are setting boolean values.  But I hadn't considered that this is an example of why it's necessary.  I guess without that function, this might not evaluate correctly (if tight wasn't in brackets)?